apps/prairielearn/assets/scripts/lib/editAccessRuleModal.ts

```diff
@@ -0,0 +1,170 @@
+            <span class="input-group-text">${timeZoneName}</span>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="end_date">End date</label>
+        <small class="form-text text-muted">
+          Only allow access before this date. All times are in the timezone of the course
+          instance.</small
+        >
+        <div class="input-group">
+          <input
+            type="datetime-local"
+            step="1"
+            class="form-control"
+            id="end_date"
+            name="end_date"
+            value="${escapeHtml(html`${accessRule.formatted_end_date}`)}"
+          />
+          <div class="input-group-append">
+            <span class="input-group-text">${timeZoneName}</span>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="active">Active</label>
+        <small class="form-text text-muted">
+          Whether the student can create a new assessment instance and submit answers to questions.
+        </small>
+        <select class="form-control" id="active" name="active">
+          <option value="True" ${accessRule.active === 'True' ? html`selected` : ''}>True</option>
+          <option value="False">False</option>
+        </select>
+      </div>
+      <div class="form-group">
+        <label for="credit">Credit</label>
+        <small class="form-text text-muted">
+          Maximum credit as percentage of full credit (can be more than 100).
+        </small>
+        <div class="input-group">
+          <input
+            type="text"
+            class="form-control"
+            id="credit"
+            name="credit"
+            value="${accessRule.credit}"
+          />
+          <div class="input-group-append">
+            <span class="input-group-text">%</span>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="time_limit">Time limit</label>
+        <small class="form-text text-muted">
+          Time limit in minutes to complete an assessment (only for Exams).
+        </small>
+        <div class="input-group">
+          <input
+            type="text"
+            class="form-control"
+            id="time_limit"
+            name="time_limit"
+            value="${accessRule.time_limit}"
+          />
+          <div class="input-group-append">
+            <span class="input-group-text">mins</span>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="password">Password</label>
+        <small class="form-text text-muted">
+          Password required to start an assessment (only for Exams).
+        </small>
+        <input
+          type="text"
+          class="form-control"
+          id="password"
+          name="password"
+          value="${accessRule.password}"
+        />
+      </div>
+      <div class="form-group">
+        <label for="pt_exam_name">PrairieTest Exam</label>
```

Comment:
```suggestion
        <label for="pt_exam_name">PrairieTest Exam UUID</label>
```

Reply:
Updated locally in [1dfef07](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/1dfef07527ed0296ff3cdfa7c58d85fce2304b97).